extensions/wikia/CuratedContent/crowdin.conf

```diff
@@ -0,0 +1,6 @@
+[crowdin.project]
+projectid = "ext-curated-content"
+format = "mediawiki"
+repo.breanch = "${projectid}-translations"
```

Comment:
Out of curiosity, whats the significance of this config line ? where is it used ?